libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -273,6 +274,22 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateCrossCloudMenu = (content) => {
+  const crossCloudMenuEl = content.querySelector('.cross-cloud-menu');
+  if (!crossCloudMenuEl) return;
+
+  const crossCloudMenuContent = toFragment`<div class="feds-crossCloudMenu"></div>`;
+
+  [...crossCloudMenuEl.querySelector('ul').children].forEach((el, index) => {
+    const decoratedLink = decorateLinkGroup(el, index);
+    if (index === 0) decoratedLink.prepend(toFragment`${CONFIG.icons.home}`);
+    crossCloudMenuContent.append(toFragment`<div class="feds-crossCloudMenu-item">${decoratedLink}</div>`);
+  });
+
+  content.append(toFragment`<div class="feds-crossCloudMenu-wrapper">${crossCloudMenuContent}</div>`);
```

Comment:
I see that the unit tests use a `selectors.crossCloudMenuWrapper`, could we not use it here as well?

Reply:
Doesn't quite work in here because `selectors.crossCloudMenuWrapper` has a dot at the beginning -> `crossCloudMenuWrapper: '.feds-crossCloudMenu-wrapper'`

Reply:
One strategy we're using in other places is simply calling `.slice(1)`